src/scan/postgres_scan.cpp

```diff
@@ -44,11 +44,11 @@ PostgresScanGlobalState::InitGlobalState(duckdb::TableFunctionInitInput &input)
 
 	if (input.CanRemoveFilterColumns()) {
 		for (duckdb::idx_t i = 0; i < input.projection_ids.size(); i++) {
-			m_projections[i] = input.column_ids[input.projection_ids[i]];
+			m_column_selection[i] = input.column_ids[input.projection_ids[i]];
 		}
 	} else {
-		for (duckdb::idx_t i = 0; i < input.projection_ids.size(); i++) {
```

Comment:
Just to be clear, this line contains the fix right? the rest of the PR is  just doing a field rename.